app/models/viewmodels/HtmlTableRow.scala

```diff
@@ -22,9 +22,14 @@ class HtmlTableRow(val label: String, val values: Seq[Option[String]])
 
 object HtmlTableRow {
 
-  def apply(label: String, value: Iterable[_]): HtmlTableRow = new HtmlTableRow(label, adjust(value))
+  def apply[T](label: String, value: Iterable[T]): HtmlTableRow = new HtmlTableRow(label, adjust(value))
 
-  private def adjust(iterable: Iterable[_]): Seq[Option[String]] = iterable match {
+  def apply[T](label: String, value: Seq[String]): HtmlTableRow = new HtmlTableRow(label, value.map(Option.apply))
+
+  def apply(label: String, value: String): HtmlTableRow = new HtmlTableRow(label, Seq(Some(value)))
+
+  // TODO review adjust method, working wrong
```

Comment:
Out of interest, how is it "wrong"?   Maybe add some detail to the TODO for next person to fix?

Reply:
We should explain what is wrong at least in the comment :)

Reply:
Yes, thank you, it's fixed right now. The method handles Option[Option[String]] but struggle with Option[String] or something like this. it's unnecessary. I removed adjust method